plugin.video.orftvthek/addon.xml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.orftvthek" name="ORF TVthek" version="0.9.2" provider-name="sofaking">
+<addon id="plugin.video.orftvthek" name="ORF TVthek" version="0.10.0" provider-name="sofaking">
     <requires>
         <import addon="xbmc.python" version="2.14.0"/>
-        <import addon="script.module.future" version="0.16.0" />
-        <import addon="script.module.parsedom" version="0.9.1"/>
-        <import addon="script.common.plugin.cache" version="0.9.1"/>
+        <import addon="script.module.future" version="0.16.0"/>
+        <import addon="script.module.kodi-six" version="0.0.5"/>
         <import addon="script.module.simplejson" version="2.0.10" optional="false"/>
     </requires>
     <extension point="xbmc.python.pluginsource" library="default.py">
```

Comment:
can you use a SPDX identifier in the license field please instead of the full name? See: https://spdx.org/licenses/
Should be `GPL-2.0-only` or `GPL-2.0-or-later`